docs/doc/30-reference/00-api/03-streaming-load.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Streaming Load API
+sidebar_label: Streaming Load API
+description:
+  Streaming Load API
+---
+
+The Streaming Load API is used to read data from your local files and load it into Databend.
+
+![image](../../../public/img/load/load-data-from-local-fs.png)
+
+## API Request Format
+
+To create a request with the Streaming Load API, follow the format below:
+
+```bash
+curl -H "<parameter>:<value>"  [-H "<parameter>:<value>"...] -F "upload=@<file_location>" -XPUT http://<user_name>:[password]@<http_handler_host>:<http_handler_port>/v1/streaming_load
+```
+## Explaining Argument `-H`
+
+The request usually includes many occurrences of the argument `-H` and each is followed by one of the following parameters to tell Databend how to handle the file you're loading data from:
+
+| Parameter               | Values                                                                                                                                                                                                                                                                                                     | Supported Formats         | Examples                                       |
+|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|------------------------------------------------|
+| insert_sql              | [INSERT_statement] + format [file_format]                                                                                                                                                                                                                                                                  | All                       | -H "insert_sql: insert into ontime format CSV" |
```

Comment:
insert_sql is required.
format_xxx is optional, and will not report an error if user spells it wrongly, since it is an ordinary HTTP header